test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -2451,6 +2451,14 @@ static int traffic_mngr_check_shaper(void)
 	odp_cpumask_t cpumask;
 	int cpucount = odp_cpumask_all_available(&cpumask);
 
+/* Skip the shaper test on arm64 systems */
+#if defined(__aarch64__)
+	ODPH_DBG("\nTemporarily skip shaper test which intermittenly "
```

Comment:
This warning should be printed always (ODPH_DBG is filtered out normally), so that it's more visible that the test case was skipped temporarily.

Reply:
Okay, I will make the change. Thanks!